public/js/sidebar.js

```diff
@@ -100,13 +103,55 @@ export default class extends Component {
         return false;
     }
 
+    /**
+     * Fixes tooltips in the sidebar, allows tooltip display on click
+     */
+    componentDidMount() {
+        $(function () {
+            $('.sidebar [data-toggle="tooltip"]').tooltip({placement: 'right'});
+            $('#copyTooltip').tooltip({title:'Copied!',trigger:'click', placement: 'right',delay: 0});
+        });
+    }
+
+    /**
+     * Handles copying the URL into the user's clipboard. Modified from: https://stackoverflow.com/a/49618964/18117380
+     * Hides the 'Copied!' tooltip after 3 seconds 
+     */
+
+    copyURL() {
+        var element = document.createElement('input'), url = window.location.href;
```

Comment:
I find multiple variable assignment on a single line quite confusing. In the minimum, break line after `,`. Better, use two `var` statements.